core/spec/lib/search/base_spec.rb

```diff
@@ -22,11 +22,12 @@
 
   context "when include_images is included in the initialization params" do
     let(:params) { { include_images: true, keyword: @product1.name, taxon: @taxon.id } }
+    let(:image_file) { File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'blank.jpg')) }
```

Comment:
As we are using `File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', **)` in several place, what do you think about extracting it to a test helper method?

Reply:
I think this is a great idea, I can certainly implement this now.